frontend/src/app/login/page.tsx

```diff
@@ -1,7 +1,74 @@
+import { useState } from "react";
+import { useRouter } from 'next/router';
+
 export default function Login() {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [error, setError] = useState("");
+    const router = useRouter();
+
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+    
+        const res = await fetch("/api/login", {
```

Comment:
Refer to the handleSubmit function in the Register page. Get username and password from the form rather than using state variables